e2e-tests/editor.spec.ts

```diff
@@ -211,3 +211,26 @@ test('copy and paste block after editing new block', async ({ page, block }) =>
 
   await expect(page.locator('text="Typed block"')).toHaveCount(1);
 })
+
+test('press escape when action menu is open, should close action menu only #6270', async ({ page, block }) => {
```

Comment:
I don't think we refer to this as an action menu. Something like page autocompletion or page completion popup probably makes more sense to the team

Reply:
I asked around on some designer chats to see if anyone knows what the heck this kind of UI element is called.

Reply:
(fwiw "menu" felt right to me because it's like a context menu, though not mouse-triggered, and "action" felt like the most fitting general term for all the things this thing does)

Reply:
One term I'm finding in search is "typeahead" or "typeahead menu", like https://github.com/twitter/typeahead.js, though this is mostly used as a term for the search results shown under search boxes. But per [its wiki page](https://en.wikipedia.org/wiki/Typeahead), I think ["Autocomplete"](https://en.wikipedia.org/wiki/Autocomplete) is actually the more accurate term for the same kind of thing, so maybe it should be autocomplete.

Aight, given all that, I'm going to call it the "autocomplete menu" since "menu" shows up in that Autocomplete wiki article describing this UI element.